docs/interpreter/drill.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: page
+title: "Apache Drill Interpreter"
+description: "How to use Apache Drill with Zeppelin"
+group: manual
+---
+{% include JB/setup %}
+
+## Drill Interpreter for Apache Zeppelin
+The [Apache Drill](http://drill.apache.org/) ™ supports a variety of NoSQL databases and file systems, including HBase, MongoDB, MapR-DB, HDFS, MapR-FS, Amazon S3, Azure Blob Storage, Google Cloud Storage, Swift, NAS and local files. A single query can join data from multiple datastores. For example, you can join a user profile collection in MongoDB with a directory of event logs in Hadoop.
+
+Drill's datastore-aware optimizer automatically restructures a query plan to leverage the datastore's internal processing capabilities. In addition, Drill supports data locality, so it's a good idea to co-locate Drill and the datastore on the same nodes.
+
+### Configuration
+
+* Copy drill-jdbc-all-<VERSION>.jar to ZEPPELIN_HOME/interpreter/jdbc/
+* Restart Zeppelin, if required.
+* Modify the JDBC Interpreter.
+You can use the JDBC Interpreter to add configurations specific to Drill.
+You need to configure the driver & the JDBC URL.
+
+<table class="table-configuration">
+  <tr>
+    <th>Property</th>
+    <th>Value</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>drill.driver</td>
+    <td>org.apache.drill.jdbc.Driver</td>
+    <td>JDBC Driver Class</td>
+  </tr>
+  <tr>
+    <td>drill.url</td>
+    <td>jdbc:drill:schema=dfs;zk=localhost:2181/drill/drillbits1</td>
+    <td>This is a sample URL. Details on Drill JDBC URL @ https://drill.apache.org/docs/using-the-jdbc-driver</td>
+  </tr>
+</table>
+
+## How to use
+Basically, you can use
+
+```%jdbc(drill)
```

Comment:
Hi @malur! Thanks for the documentation : )

Zeppelin is using [jekyll](https://github.com/apache/incubator-zeppelin/tree/master/docs#build-documentation) when it builds the website. But when it renders this markdown docs to html page, an error is occured because of the `%jdbc(drill)` which is next to the **```** as below.
![screen shot 2016-02-16 at 12 40 50 am](https://cloud.githubusercontent.com/assets/10060731/13053147/45523310-d446-11e5-97de-86b892ee22e2.png)

So, could you add a new line between **```** and `%jdbc(drill)` ?